dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/QuayImageRegistry.java

```diff
@@ -107,20 +115,33 @@ public QuayImageRegistry(final Token quayToken) {
         final List<Tag> tags = new ArrayList<>();
         final Optional<QuayRepo> toolFromQuay = getToolFromQuay(tool);
         if (toolFromQuay.isPresent()) {
-            final QuayRepo quayRepo = toolFromQuay.get();
-            final Map<String, QuayTag> tagsFromRepo = quayRepo.getTags();
-            final int maxQuayTagsReturnedByRepo = 500;
-            List<QuayTag> quayTags = new ArrayList<>(tagsFromRepo.values());
-            if (tagsFromRepo.size() == maxQuayTagsReturnedByRepo) {
-                try {
-                    quayTags = getAllQuayTags(repo);
-                } catch (ApiException e) {
-                    throw new CustomWebApplicationException("Could not get QuayTag", HttpStatus.SC_INTERNAL_SERVER_ERROR);
-                }
+            List<QuayTag> quayTags;
+            try {
+                quayTags = getAllQuayTags(repo);
+            } catch (ApiException e) {
+                throw new CustomWebApplicationException("Could not get QuayTag", HttpStatus.SC_INTERNAL_SERVER_ERROR);
```

Comment:
It was like this, but "QuayTag" should be, I think "Quay tag" or maybe "Quay Tag".